components/automate-chef-io/content/docs/on-prem-builder.md

```diff
@@ -0,0 +1,93 @@
++++
+title = "Deploying Builder with Chef Automate"
+description = "use Chef Automate to deploy Builder"
+date = 2019-11-19T14:10:15-08:00
+weight = 20
+draft = false
+bref = ""
+toc = true
++++
+
+## Overview
+Enterprise customers may wish to set up an on-premises Habitat Builder depot to store
+Habitat packages for use by their own Habitat Studios and Supervisors. Use Chef Automate
+to deploy and manage an on-premises Habitat Builder depot.
+
+{{< warning >}}
+The current implementation of Automate-deployed Habitat Builder is an early milestone
+intended for use in proof-of-concept deployments. We do not consider it a fully-mature
+feature, and you should not either.
+{{< /warning >}}
+
+### System requirements
+Currently, Habitat Builder requires Chef Automate to be installed on the same host for
+authentication purposes. Outside the boundaries of a proof-of-concept exercise, we
+recommend against running installations of Chef Automate and Habitat Builder on
+the same host.
+
+#### Hardware
+The minimum with which Chef Automate and Habitat Builder can be deployed on a single host
+is:
+
```

Comment:
The following numbers are from merging the numbers from

https://automate.chef.io/docs/system-requirements/

and 

https://github.com/habitat-sh/on-prem-builder/blob/master/on-prem-docs/content/docs/getting-started.md#requirements

If someone wants to suggest something else, that would be fine.

Reply:
I added #2353 for adjusting the Automate preflight checks to take Builder into account.